src/request/request.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import { NetworkError } from './errors'
+
+const CORE_API = 'https://api.core.ac.uk/internal'
+// TODO: Add package version to UA
+const USER_AGENT = '@oacore/api'
+
+const apiRequest = async (url, method = 'GET', params = {}) => {
+  try {
+    const response = await axios({
+      url: `${CORE_API}${url}`,
+      method,
+      params,
+      headers: {
+        'Content-Type': 'application/json',
+        'User-Agent': USER_AGENT,
```

Comment:
I am not sure it's a good idea. We should keep in mind that any header may be redefined in any request, e.g. in CURL. So we basically remove UA analytics on API side by this line, even if we do not have any.